### PR TITLE
[B2BP-748] - Remove S3 CDN ECS and IAM policy for old Strapi

### PR DESCRIPTION
**Pull request by maurodandrea**
Remove bucket S3,  CDN, ECS and IAM policy for old Strapi

Depends on #345 

#### List of Changes
cdn.tf s3_bucket.tf ecs.tf and iam_policy.tf files.

#### Motivation and Context
Remove bucket S3,  CDN, ECS and IAM policy for old Strapi

#### Types of changes

- [X] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [X] My change not requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/infrastructure/src/iam_policy.tf [hunk 1]
```diff
@@ -100,7 +85,6 @@ data "aws_iam_policy_document" "ecs_task_role_s3" {
       "s3:PutObjectAcl"
     ]
     resources = concat(
-      [aws_s3_bucket.cms_medialibrary_bucket.arn],
       [for name, bucket in aws_s3_bucket.cms_multitenant_medialibrary_bucket : bucket.arn]
     )
```
Comment (christian-calabrese) [inline apps/infrastructure/src/iam_policy.tf:89]: The concat can now be removed by only leaving the list comprehension at line 88

File: apps/infrastructure/src/iam_policy.tf [hunk 2]
```diff
@@ -142,7 +126,6 @@ data "aws_iam_policy_document" "ecs_task_execution" {
       "s3:GetBucketLocation"
     ]
     resources = concat(
-      [aws_s3_bucket.cms_medialibrary_bucket.arn],
       [for name, bucket in aws_s3_bucket.cms_multitenant_medialibrary_bucket : bucket.arn]
     )
```
Comment (christian-calabrese) [inline apps/infrastructure/src/iam_policy.tf:130]: Same as above

File: apps/infrastructure/src/iam_policy.tf [hunk 3]
```diff
@@ -250,7 +233,6 @@ resource "aws_iam_policy" "upload_image" {
         ]
         Effect = "Allow"
         Resource = concat(
-          ["${aws_s3_bucket.cms_medialibrary_bucket.arn}/*"],
           [for name, bucket in aws_s3_bucket.cms_multitenant_medialibrary_bucket : "${bucket.arn}/*"]
         )
```
Comment (christian-calabrese) [inline apps/infrastructure/src/iam_policy.tf:237]: Same as above

File: apps/infrastructure/src/iam_policy.tf (hunk 1 above)
Comment (maurodandrea) [reply apps/infrastructure/src/iam_policy.tf]: concat removed with last commit

File: apps/infrastructure/src/iam_policy.tf (hunk 2 above)
Comment (maurodandrea) [reply apps/infrastructure/src/iam_policy.tf]: concat removed with last commit

File: apps/infrastructure/src/iam_policy.tf (hunk 3 above)
Comment (maurodandrea) [reply apps/infrastructure/src/iam_policy.tf]: concat removed with last commit

**Review by christian-calabrese — APPROVED** [review]
LGTM

**Pull request merged**